answers/Java/@thecarisma/N04TypeInReverse.java

```diff
@@ -0,0 +1,54 @@
+
+import java.util.Random;
+import java.util.Scanner;
+
+/*
+ * The MIT License
+ *
+ * Copyright 2019 Azeez Adewale <azeezadewale98@gmail.com>.
+ *
+ */
+
+/**
+ *
+ * @author Azeez Adewale <azeezadewale98@gmail.com>
+ */
+public class N04TypeInReverse {
+    
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+        String word, userInput ;
+        
+        while (true) {
+            word = getRandomWord();
+            System.out.printf("Type the word '%s' in reverse: ", word);
+            userInput = scanner.nextLine();
+            if (userInput.equals(reverseWord(word))) 
+                System.out.println("✅");
```

Comment:
I get an error saying `"error: unmappable character (0x9D) for encoding windows-1252"`

See https://stackoverflow.com/a/464886/7396801